src/components/Chart/Chart.js

```diff
@@ -148,7 +152,14 @@ const Chart = ({ title, hint, chartData }) => {
               },
               tooltips: {
                 callbacks: {
-                  title: () => null,
+                  title: (item) => {
+                    const [month, year] = item[0].label.split("/");
+                    return `${months[month - 1]} ${2000 + toInt(year)}`;
```

Comment:
Isn't the year that's coming in via the data.json file already 4 digits? Why would we need to do this? If it's stripped of the left-most 2 digits at some point for some reason, we should consider changing that instead. It's conceivable that we'll have data that dates back to before 2000 and this logic would generate years like 2096 and 2099.

Reply:
@jessex Good point, fixed